From 3f6c93cecfd3a92b8fad77032a489279391d8a1e Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 19 Sep 2021 19:26:29 +0000
Subject: [PATCH 23/52] Revert "SystemUI: Don't show FOD icon/pad keyguard if
 strong auth is required"

This reverts commit b10a11936f11bfbf79bf0c98bd27dd8188ce2f80.
---
 .../com/android/keyguard/KeyguardSecurityContainer.java    | 7 +++----
 .../src/com/android/systemui/biometrics/FODCircleView.java | 5 -----
 2 files changed, 3 insertions(+), 9 deletions(-)

diff --git a/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java b/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
index a8f15a0acc0..bace9324ac9 100644
--- a/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
+++ b/packages/SystemUI/src/com/android/keyguard/KeyguardSecurityContainer.java
@@ -525,10 +525,9 @@ public class KeyguardSecurityContainer extends FrameLayout implements KeyguardSe
 
         // Consume bottom insets because we're setting the padding locally (for IME and navbar.)
         int inset;
-        int minBottomMargin = mHasFod && !mUpdateMonitor.userNeedsStrongAuth() &&
-                mUpdateMonitor.isFingerprintDetectionRunning() ?
-                        getResources().getDimensionPixelSize(
-                                R.dimen.kg_security_container_min_bottom_margin) : 0;
+        int minBottomMargin = mHasFod && mUpdateMonitor.isFingerprintDetectionRunning() ?
+                getResources().getDimensionPixelSize(
+                        R.dimen.kg_security_container_min_bottom_margin) : 0;
 
         if (sNewInsetsMode == NEW_INSETS_MODE_FULL) {
             int bottomInset = insets.getInsetsIgnoringVisibility(systemBars()).bottom;
diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
index ce4b9ca2789..5a5f2914d14 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
@@ -521,11 +521,6 @@ public class FODCircleView extends ImageView implements TunerService.Tunable {
     }
 
     public void show() {
-        if (mUpdateMonitor.userNeedsStrongAuth()) {
-            // Keyguard requires strong authentication (not biometrics)
-            return;
-        }
-
         if (!mUpdateMonitor.isScreenOn() && !mFodGestureEnable) {
             // Keyguard is shown just after screen turning off
             return;
-- 
2.30.2

